docs/docs/responses.mdx

```diff
@@ -140,22 +140,16 @@ in the shell and in interactive learning.
 :::
 
 
-### Conditional Response Variations (experimental)
-
-:::caution
-This feature is currently experimental and might change or be removed in the future.
-Share your feedback on it in the [forum](https://forum.rasa.com) to help
-us make this feature ready for production.
-:::
+### Conditional Response Variations
 
 Specific response variations can also be selected based on one or more slot values using a conditional
 response variation. A conditional response variation is defined in the domain or responses YAML files
 similarly to a standard response variation but with an additional `condition` key. This key specifies
 a list of slot `name` and `value` constraints.
 
 :::note
-Because the feature is currently experimental, it is recommended to set the `influence_conversation`
-slot setting to `False` for the slot mappings which are part of conditions.
+It is recommended to set the `influence_conversation` slot setting to `False` for the slot mappings
+which are part of conditions.
 :::
```

Comment:
Do we want to keep this recommendation? I have gotten several questions about it from CSEs and customers, and it seems to confuse people, and I just end up telling people that its just a recommendation and they can go ahead and try using featurized slots in CRV constraints. I haven't heard of anyone having a subsequent issue with that.

Reply:
@ancalita Wdyt?

Reply:
I guess if no issues arose from using CRVs with featurized slots, it's fine to remove.

Reply:
Okay. I removed it